src/ugeneui/src/app_settings/user_apps_settings/UserApplicationsSettingsGUIController.cpp

```diff
@@ -111,7 +113,15 @@ UserApplicationsSettingsPageWidget::UserApplicationsSettingsPageWidget(UserAppli
         it.next();
         langCombo->addItem(it.key(), it.value());
     }
-    styleCombo->addItems(QStyleFactory::keys());
+    auto keys = QStyleFactory::keys();
+    for (const auto& key : keys) {
+        if (STYLE_KEYS_FIXED_REGISTER.keys().contains(key)) {
```

Comment:
To you really need to create  keys() set to check of the map contains a key?

Reply:
Removed